vm/interpreter/src/gas_tracker/mod.rs

```diff
@@ -2,7 +2,17 @@
 // SPDX-License-Identifier: Apache-2.0, MIT
 
 mod gas_charge;
-mod price_list;
 
 pub use self::gas_charge::GasCharge;
-pub use self::price_list::{price_list_by_epoch, PriceList};
+use fvm::gas::{price_list_by_network_version, PriceList};
+use fvm_shared::{clock::ChainEpoch, version::NetworkVersion};
+
+/// Returns gas price list by Epoch for gas consumption.
+pub fn price_list_by_epoch(epoch: ChainEpoch, calico_height: ChainEpoch) -> &'static PriceList {
+    let version = if epoch < calico_height {
+        NetworkVersion::V15
+    } else {
+        NetworkVersion::V0
+    };
+    price_list_by_network_version(version)
+}
```

Comment:
Remove this function and update all callers. They should already have a network version or a means to get the network version from the epoch.

Reply:
Removed